### PR TITLE
feat: add tooltip to show multiple tables for dashboard filters

### PR DESCRIPTION
**Pull request by IrakliJani**
Closes: https://github.com/lightdash/lightdash/issues/3660

### Description:
shows multiple tables 

CleanShot 2023-01-03 at 18 02 09

### CODE REVIEW

File: packages/frontend/src/components/common/Filters/configs.ts [hunk 1]
```diff
@@ -165,3 +169,19 @@ export const getFilterRuleLabel = (
...
 };
+
+export const getFilterRuleTables = (
+    filterRule: FilterRule | DashboardFilterRule,
+    field: FilterableField,
+): string[] => {
+    if (isDashboardFilterRule(filterRule) && filterRule.tileTargets) {
+        return Object.values(filterRule.tileTargets).reduce<string[]>(
+            (tables, tileTarget) => {
+                return uniq([...tables, capitalize(tileTarget.tableName)]);
```
Comment (IrakliJani) [inline packages/frontend/src/components/common/Filters/configs.ts:180]: should I be using table label here instead of table name?

Comment (ZeRego) [reply packages/frontend/src/components/common/Filters/configs.ts:180]: Yes, the table name is only used as an ID.

**Review by ZeRego — APPROVED** [review]
Code looks good 👍 just needs to se the table label instead of the table name.

File: packages/frontend/src/components/common/Filters/configs.ts (hunk 1 above)
Comment (ZeRego) [reply packages/frontend/src/components/common/Filters/configs.ts:180]: Let's not capitalize the label. The label was defined by the user or automatically generated via the table name using the `friendlyName()` function that capitalizes it.

Comment (IrakliJani) [reply packages/frontend/src/components/common/Filters/configs.ts]: @ZeRego 👍 updated

**Pull request merged**